features/assignment/random.feature

```diff
@@ -12,9 +12,9 @@ Feature: Random Assignment.
     And `v3` is assigned a random string
     When the response body is assigned `[ "{{ v1 }}","{{ v2 }}","{{ v3 }}" ]`
     Then the value of the response body does not have any element that is empty
-    And the value of the response body child `"[0]"` is equal to `{{ v1 }}`
-    And the value of the response body children `"[1:2]"` does not have any element that is equal to `{{ v1 }}`
-    And the value of the response body child `"[2]"` is not equal to `{{ v2 }}`
+    And the value of the response body child `.[0]` is equal to `{{ v1 }}`
```

Comment:
Adopting a better convention around a cleanup/documentation merge earlier, not related to primary purpose.